### PR TITLE
Schedules paging, day routes

### PR DESCRIPTION
**Pull request by sjmcculloch**
- Added paging between weeks for availability view
- Moved some availability logic to shared selectors
- Added routes & linked in ScheduleDayView
- Added new TableNextPrevious component

image

### CODE REVIEW

**Comment by SSimong** [comment]
if you guys could check and approve this PR it would be great for the demos this week!

**Comment by sdunster** [comment]
its approved - on scott to merge now

File: web/src/components/Tables/TableNextPrevious.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import PropTypes from 'prop-types';
+
+import { withStyles } from 'material-ui/styles';
+import styles from './TableNextPrevious.styles';
+
+const TableNextPrevious = ({ classes, children, hasNext, hasPrevious, pressNext, pressPrevious }) =>
+  hasNext &&
+  hasPrevious && (
+    <div className={classes.navPanel}>
+      <div>{hasPrevious && <button onClick={pressPrevious}>Previous</button>}</div>
```
[the remark this answers is not in the document]
Comment (sjmcculloch) [reply web/src/components/Tables/TableNextPrevious.js]: Yes I changed this to completely hide the div, it should be a (hasNext || hasPrevious) && (output) - I'll update it

**Comment by sjmcculloch** [comment]
Merged in @SSimong

**Pull request merged**